xarm_controller/src/hardware/uf_robot_system_hardware.cpp

```diff
@@ -356,6 +356,15 @@ namespace uf_robot_hardware
         }
         initialized_ = true;
         
+        // Warnings will fly when below 80Hz
+        const auto max_period = 0.0125;
```

Comment:
Any way to get the desired update period so we don't need to hard code this?

Reply:
Not at this time, unfortunately. Same situation as with the kortex driver. It's defined by us ad-hoc